src/cluster.c

```diff
@@ -1594,6 +1594,13 @@ void resetClusterStats(void) {
     if (!server.cluster_enabled) return;
 
     clusterSlotStatResetAll();
+
+    if (server.cluster) {
+        for (int i = 0; i < CLUSTERMSG_TYPE_COUNT; i++) {
+            server.cluster->stats_bus_messages_sent[i] = 0;
+            server.cluster->stats_bus_messages_received[i] = 0;
+        }
+    }
```

Comment:
i don't think we need this server.cluster check, the structure should be ready in this moment. Also clusterSlotStatResetAll don't check it.
```
void clusterSlotStatResetAll(void) {
    memset(server.cluster->slot_stats, 0, sizeof(server.cluster->slot_stats));
}
```